src/args.ts

```diff
@@ -83,7 +84,7 @@ export function parseArgs() {
     configuration: {
       "populate--": true,
       "strip-dashed": true,
-    }
+    },
```

Comment:
Why does this show up here now? Are we not using Prettier in lage?

Reply:
no Prettier hasn't been configured for lage. Will remove the trailing comma. Prettier can be added in a separate PR